packages/react-components/src/StakingInputExtrinsic/options/method.tsx

```diff
@@ -8,7 +8,7 @@ import React from 'react';
 import { ApiPromise } from '@polkadot/api';
 
 export default function createOptions (api: ApiPromise, sectionName: string): DropdownOptions {
-  let section = api.tx[sectionName];
+  let section = {...api.tx[sectionName]};
```

Comment:
donno how but this was missing in this branch, but available in manage_stake_page branch at -https://github.com/cennznet/ui/blob/93bc2d0d7085cd08a3460a4a99c31aaca805fff7/packages/react-components/src/StakingInputExtrinsic/options/method.tsx#L11